src/components/common/Input/Input.jsx

```diff
@@ -39,4 +39,4 @@ Input.propTypes = {
   error: PropTypes.string,
 };
 
-export default Input;
```

Comment:
I got this error while testing this PR: 

<img width="749" alt="error" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112666943/0c81da15-2bb4-4f4e-a3ea-58d2ab7a9e35">

But if you comment "import './input.css" line in the file src/components/common/Input/Input.jsx. The functionality will work as expected.

<img src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112666943/946728d9-7e79-4ae0-b98d-fa4f2f5d895c" alt="end date" />
<img src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112666943/301b8e54-874c-4960-8d00-e46d8ce1b1ad" alt="timer" />